src/psyclone/transformations.py

```diff
@@ -1361,3 +1363,83 @@ def apply(self, loop, depth=None):
         loop.update_halo_exchanges()
 
         return schedule, keep
+
+
+class LoopSwapTrans(Transformation):
+    ''' Provides a loop-swap transformation.
+        For example:
+
+        >>> from parse import parse
+        >>> from psyGen import PSyFactory
+        >>> ast,invokeInfo=parse("dynamo.F90")
+        >>> psy=PSyFactory("dynamo0.1").create(invokeInfo)
+        >>> schedule=psy.invokes.get('invoke_v3_kernel_type').schedule
+        >>> schedule.view()
+        >>>
+        >>> from transformations import LoopSwapTrans
+        >>> swap=LoopSwapTrans()
+        >>> new_schedule,memento=swap.apply(schedule.children[0],
```

Comment:
Please update this example to match the API of the `apply` method - i.e. it only takes a single Node as argument.